app/utils/stripe.js

```diff
@@ -0,0 +1,45 @@
+/* globals Stripe */
+
+import jQuery from 'jquery';
+import Ember from 'ember';
+import config from '../config/environment';
+
+var loadStripe;
+var stripeMock = {};
+
+if(config.env === 'test') {
+  loadStripe = Ember.RSVP.resolve;
+  createStripeToken = Ember.RSVP.resolve;
```

Comment:
Maybe it would be better to export an object here, and use `loadStripe` internally:

```javascript
var loadPromise;
var loadStripe = function(){
  if (!loadPromise) { loadPromise = Ember.RSVP.Promise(/* jQuery.getScript, etc, returning the Stripe global */); }
  return loadPromise;
};
export default {
  createToken: function(options){
    return loadStripe().then(function(stripe){
       return stipe.card.createToken(options);
    });
  }
}
```

Then in tests you could `import Stripe from "../utils/stripe"` and stub out `Stripe.createToken` to be a resolving or rejecting promise, depending on what you are testing. ?